shared/src/commonMain/kotlin/vn/luongvo/kmm/survey/data/remote/datasource/AuthRemoteDataSource.kt

```diff
@@ -0,0 +1,25 @@
+package vn.luongvo.kmm.survey.data.remote.datasource
+
+import io.ktor.client.request.*
+import io.ktor.http.*
+import kotlinx.coroutines.flow.Flow
+import vn.luongvo.kmm.survey.data.remote.apiclient.ApiClient
+import vn.luongvo.kmm.survey.data.remote.apiclient.builder.path
+import vn.luongvo.kmm.survey.data.remote.body.LoginRequestBody
+
+interface AuthRemoteDataSource {
+    fun logIn(body: LoginRequestBody): Flow<String>
```

Comment:
Should we add todo for return type or we will return String?

Reply:
@blyscuit As I said, I just added the raw String response handling to verify the network layer implementation. There are several places with String data type instead of a Login result or Token model, so I think we can keep them as they are. I will do the rest of the tasks in this ticket https://github.com/luongvo/kmm-survey/issues/7, don't worries.